src/taskgraph/transforms/run/__init__.py

```diff
@@ -252,7 +251,18 @@ def use_fetches(config, tasks):
         for kind in sorted(fetches):
             artifacts = fetches[kind]
             if kind in ("fetch", "toolchain"):
-                for fetch_name in sorted(artifacts):
+                for artifact in sorted(artifacts):
+                    # Convert name only fetch entries to full fledged ones for
+                    # easier processing.
+                    if isinstance(artifact, str):
+                        artifact = {
+                            "artifact": artifact,
+                            "dest": None,
+                            "verify-hash": False,
+                            "extract": True,
```

Comment:
Those are the defaults so maybe don't specify them here?

Reply:
ah, true